### PR TITLE
[SDK-10] Responsive UI

### PR DESCRIPTION
**Pull request by srcnalt**
## Description

- UI panels and prefabs are updated to correctly align in wide and narrow screens.

#### Updated
- AvatarCreatorExample Sample project updated.
- AvatarSelection, FullbodyButton, and DefaultAvatarButton prefabs are updated.

## How to Test

- Import RPM SDK core as the first step
- Import Custom Avatar Creator package via https://github.com/readyplayerme/rpm-unity-sdk-avatar-creator.git#feature/ui-updates
- Import the Avatar Creator Sample project
- Set the screen to Free Aspect from Game Window
- Stretch the game window into narrow and wide sizes in every editor screen to observe clamping UI elements.

## Checklist

-   [ ] Tests written or updated for the changes.
-   [ ] Documentation is updated.
-   [x] Changelog is updated.
-   [ ] QA Testing is updated.

### CODE REVIEW

**Comment by srcnalt** [comment]
@rYuuk @HarrisonHough no code rev is needed, I merged dev into this branch. Just prefabs and scene is updated, stretch it around and see if everything is in order.

File: Samples~/Prefabs/Panels/FaceTypeAssetPanel.prefab [hunk 1]
```diff
@@ -29,17 +29,18 @@ RectTransform:
   m_LocalRotation: {x: -0, y: -0, z: -0, w: 1}
   m_LocalPosition: {x: 0, y: 0, z: 0}
   m_LocalScale: {x: 1, y: 1, z: 1}
+  m_ConstrainProportionsScale: 0
```
Comment (rYuuk) [inline Samples~/Prefabs/Panels/FaceTypeAssetPanel.prefab:32]: This coming from project being opened in unity 2021, I would suggest opening project in 2020.3 and removing all of them from all the prefabs and scene. Unity would do it on its own if you open it in 2020.3.

**Review by rYuuk — APPROVED** [review]
🟢 Thanks for fixing UI, it also gave me understanding of correct usage of anchors. Please take a look into the suggestion for changing it to the correct minimum supported unity version.
 
image

**Comment by rYuuk** [comment]
I also found the fix for null exception happening intermittently, please could you add these line 
```C#
 var colors = await LoadColors(); 
 assetButtonCreator.CreateColorUI(colors, UpdateAvatarColor);
 ```
 instead of this
 https://github.com/readyplayerme/rpm-unity-sdk-avatar-creator/blob/44c3cb53620fecc2491191facdbf8f323a2442e1/Samples~/Scripts/UI/SelectionScreens/AvatarCreatorSelection.cs#L125

**Review by HarrisonHough — APPROVED** [review]
Minor thing but perhaps it could be adjusted so that for portrait mode this would show more default avatar options? 
As it doesn't really make sense to waste all that extra screen space. 
image

Same with this color pallete panel (left) 

image

It could be extended to make better use of the extra screen space 

More of a UX issue but if you click "sign in with rpm" it's impossible to get back if you don't (or fail to) sign in. 
image

**Comment by srcnalt** [comment]
> Minor thing but perhaps it could be adjusted so that for portrait mode this would show more default avatar options?

Yep didn't go into too much responsiveness tbh, in both very wide and very narrow screen cases there will be empty spaces. Main focus was to make items not stretch.

**Pull request merged**